src/Hpack.hs

```diff
@@ -132,6 +132,7 @@ data Status =
     Generated
   | ExistingCabalFileWasModifiedManually
   | AlreadyGeneratedByNewerHpack
+  | GeneratedByNewerHpackButUnchanged
```

Comment:
I'm tempted to just treat this as `OutputUnchanged` and not add a new constructor here.

If we decide to add the new constructor, we will also need corresponding PRs for `stack`, `cabal2nix` and `tinc`.

Reply:
Makes sense. Let's go with no new constructor, since I'm not sure any API consumers will care about the difference.